<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleSIVPage</class>
 <widget class="QWidget" name="VehicleSIVPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	font: 20px, &quot;微软雅黑&quot;;
	color:white;
background-color:blue;
}
QPushButton
{
	background-color:brown ;
	border-top: 2px solid white;	
	border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="LOADVM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADVTC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCVTC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCVM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>205</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADVTC1lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>ACM直流电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADAM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADAM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCATC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADATC1lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADATC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>190</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>365</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCVM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_8">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>125</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCVTC1lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>285</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>ACM直流电压(V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LOADVM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>ACM负载电流(A)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCATC1lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>ACM负载电压(V)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCAM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="DCAM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>350</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_13">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>560</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_14">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMTC1lbl">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_15">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMM2lbl">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_16">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMTC2lbl">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_17">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>2</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ACMKMlbl">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>65</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/ACMKMclose.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SCTC2lbl">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SCM1lbl">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>潜在短路故障</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SCTC1lbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_18">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>445</y>
     <width>681</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="SCM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPageBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font:30px;
color:black;
background-color:orange;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>﹀</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_19">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;
border-radius:10px;
font:14px;
color:yellow;
background-color:darkgray;</string>
   </property>
   <property name="text">
    <string>






第1页/共2页</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl1_13</zorder>
  <zorder>lbl1_9</zorder>
  <zorder>lbl1_8</zorder>
  <zorder>lbl1_11</zorder>
  <zorder>lbl1_18</zorder>
  <zorder>lbl1_10</zorder>
  <zorder>lbl1_19</zorder>
  <zorder>LOADVM2lbl</zorder>
  <zorder>LOADVTC2lbl</zorder>
  <zorder>DCVTC2lbl</zorder>
  <zorder>DCVM2lbl</zorder>
  <zorder>LOADVTC1lbl</zorder>
  <zorder>lbl1_5</zorder>
  <zorder>LOADAM2lbl</zorder>
  <zorder>LOADAM1lbl</zorder>
  <zorder>DCATC2lbl</zorder>
  <zorder>LOADATC1lbl</zorder>
  <zorder>LOADATC2lbl</zorder>
  <zorder>DCVM1lbl</zorder>
  <zorder>DCVTC1lbl</zorder>
  <zorder>lbl1_4</zorder>
  <zorder>LOADVM1lbl</zorder>
  <zorder>lbl1_3</zorder>
  <zorder>DCATC1lbl</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>DCAM1lbl</zorder>
  <zorder>DCAM2lbl</zorder>
  <zorder>lbl1_14</zorder>
  <zorder>ACMTC1lbl</zorder>
  <zorder>ACMM1lbl</zorder>
  <zorder>lbl1_15</zorder>
  <zorder>ACMM2lbl</zorder>
  <zorder>lbl1_16</zorder>
  <zorder>ACMTC2lbl</zorder>
  <zorder>lbl1_17</zorder>
  <zorder>ACMKMlbl</zorder>
  <zorder>SCTC2lbl</zorder>
  <zorder>SCM1lbl</zorder>
  <zorder>lbl1_7</zorder>
  <zorder>SCTC1lbl</zorder>
  <zorder>SCM2lbl</zorder>
  <zorder>NextPageBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
